workspaces/javascript-leetcode-month/problems/2635-apply-transform-over-each-element-in-array/solution.md

```diff
@@ -201,6 +201,18 @@ function map<TIn, TOut>(
 }
 ```
 
+A `for` loop mutating the input array:
```

Comment:
I'd probably suggest to add this at the bottom of the section, with a "btw since the output is the same size as the input you could modify it if you wanted to save some space".

Reason being, and I was mentioning this in the Sort By write-up as well, I recommend APIs that either mutate their input or return a mutated copy, not both, so we're kinda going with a less-than-ideal API imo. (It's unavoidable, though, since LeetCode requires us to return it.)